shell/platform/darwin/ios/framework/Source/VsyncWaiterIosTest.mm

```diff
@@ -30,6 +32,25 @@ @interface VsyncWaiterIosTest : XCTestCase
 
 @implementation VsyncWaiterIosTest
 
+- (void)testSetAllowPauseAfterVsyncCorrect {
+  auto thread_task_runner = CreateNewThread("VsyncWaiterIosTest");
+  auto callback = [](std::unique_ptr<flutter::FrameTimingsRecorder> recorder) {};
```

Comment:
nit: just put this inline

Reply:
What's meaning of `put the inline`?   Cound you give me a code example?😄

Reply:
This should work:
```objc++
VSyncClient* vsyncClient = [[[VSyncClient alloc] initWithTaskRunner:thread_task_runner
                                                             callback:[] (recorder) {}] autorelease];
```

Reply:
Done.

Reply:
Oh I was thinking type inference would let you avoid the type for the parameter, that didn't work?  I forget when jumping around all these different languages.  Thanks anyway.

Reply:
Yes, I should add the type to let it work(for `recorder`). Or it will raise error when complie ^_^